reV/bespoke/cli_bespoke.py

```diff
@@ -274,7 +297,10 @@ def valid_config_keys():
               'the turbine rotor diameter.')
 @click.option('--ga_kwargs', '-gakw', type=STR, default=None,
               show_default=True,
-              help='Dictionary of keyword arguments to pass to GA initialization.')
+              help='Dictionary of keyword arguments to pass to GA '
+              'initialization.If `None`, default initialization values are '
+              'used. See :class:`~reV.bespoke.gradient_free.GeneticAlgorithm` '
```

Comment:
```suggestion
              'used. See :class:`~reV.bespoke.gradient_free.GeneticAlgorithm` '
```
```suggestion
              'used. See reV.bespoke.gradient_free.GeneticAlgorithm '
```
My bad here - the tags I threw in front of that were for sphinx documentation purposes only and probably aren't too useful for a cli user :p

Reply:
fixed